transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoop.java

```diff
@@ -46,6 +47,13 @@
     private static final AtomicIntegerFieldUpdater<EpollEventLoop> WAKEN_UP_UPDATER =
             AtomicIntegerFieldUpdater.newUpdater(EpollEventLoop.class, "wakenUp");
 
+    // Just to make it easier to switch between NIO and EPOLL.
+    private static final long MAX_SELECT_TIMEOUT =
+            SystemPropertyUtil.getLong("io.netty.maxSelectTimeout", Long.MAX_VALUE);
+
+    private static final long MAX_EPOLL_WAIT_TIMEOUT =
+            SystemPropertyUtil.getLong("io.netty.maxEpollWaitTimeout", MAX_SELECT_TIMEOUT);
```

Comment:
@trustin I guess we could just use `io.netty.maxSelectTimeout` all the time if you prefer. WdYT ?